cmd/bnscli/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/iov-one/weave/tmtest"
+)
+
+// taken from testdata/config/config.toml - rpc.laddr
+const tmURL = "http://localhost:44444"
+
+// privKeyHex is a hex-encoded private key of an account with tokens on the test server
+const privKeyHex = "d34c1970ae90acf3405f2d99dcaca16d0c7db379f4beafcfdf667b9d69ce350d27f5fb440509dfa79ec883a0510bc9a9614c3d44188881f0c5e402898b4bf3c9"
+
+// addr is the hex address of the account that corresponds to privKeyHex
```

Comment:
Those comments! :medal_sports:

Reply:
After looking at ethereum codebase, you get thankful for these :smile:

Reply:
![](https://cdn.dribbble.com/users/374643/screenshots/3241775/render.gif)